experimental/batchmap/ctmap/cmd/build/build.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 Google LLC
```

Comment:
Missing the end of the first line of the license?

Reply:
Done.